editor/src/document/document_message_handler.rs

```diff
@@ -41,6 +41,7 @@ pub struct DocumentMessageHandler {
 	artboard_message_handler: ArtboardMessageHandler,
 	#[serde(skip)]
 	transform_layer_handler: TransformLayerMessageHandler,
+	pub overlays_visible: bool,
```

Comment:
Why not store this in the overlay manager?

Reply:
I started with that approach but changed it to this because we'd ideally like to not serialize it (i.e. `#[serde(skip)]`). It's a transient thing, not a part of the document, so it doesn't belong in the file written to disk. However the options bar document state should be saved per-document, and we should eventually move towards those all being grouped into a struct.